### PR TITLE
fontWeight Token added in Typography Section

### PR DESCRIPTION
**Pull request by Bismay5467**
**Feature implemented : #195** 
Hey @alexdanilowicz ! Can you please review this pull request ?
The intention of this pull request is to add the `fontWeight` token in the Typography section.

### CODE REVIEW

**Comment by Teddarific** [comment]
Hey @Bismay5467! 

Awesome stuff here, just some high level comments:

1. We should probably let users define their own custom names for font weights, e.g. being able to rename "hairline" to whatever they want
2. I don't think Font Weights should be tied to Font Sizes, i.e. Font Weights should be a separate object to let people mix-and-match!

**Comment by Teddarific** [comment]
@Bismay5467  I think a new section underneath `Font Sizes` would be ideal! Something like this:

image

The motivation behind this is that we want to provide a place for people to simply define values that they can choose how to mix-and-match them. We have plans for a theme editor where people can then use these to define bigger themes if that makes sense!

**Comment by alexdanilowicz** [comment]
@Teddarific — thanks for chiming in. @Bismay5467 apologies that when were discussing in the issue, I didn't mention that font weight should be its own object (and NOT be tied in with fontsizes). That's on me and I apologize for the confusion.

I think what you have is a great start,  just need to make it its own object/section. Let us know if @Teddarific helped clarify it.

**Comment by alexdanilowicz** [comment]
@Bismay5467 I'm a bit confused by the merge conflicts here? Why did the docs change and looks like the dashboard file was modified in main. Are you able to clean up the merge conflicts? Thanks!

File: docs/home/export-formats/colors.mdx [hunk 1]
```diff
@@ -5,46 +5,46 @@ description: 'After configuring your theme in the editor, you can export colors
...
 
 ```javascript
 <button style={{ backgroundColor: 'var(--color-primary)'}}>
 ```
 
-## Using SCSS
-
-```scss
+````scss SCSS
+```scss SCSS
```
Comment (alexdanilowicz) [inline docs/home/export-formats/colors.mdx:22]: None of the docs should change

File: packages/core/src/components/Dashboard/index.tsx [hunk 2]
```diff
@@ -76,7 +78,7 @@ export function Dashboard({
         }
 
         setColors(data.tokens.colorData ?? [])
-        setTypography(data.tokens.typography)
```
Comment (alexdanilowicz) [inline packages/core/src/components/Dashboard/index.tsx:79]: In main, the Dashboard index file was refactored. so this needs to be updated with what's on main

File: packages/core/src/components/Onboarding/pages/ExportSettings.tsx [hunk 3]
```diff
@@ -35,7 +35,7 @@ export function ExportSettings({
   const shades = generateDefaultColorShades(primaryColor)
 
   return (
-    <Box css={{ display: 'flex', height: '100%' }}>
+    <Box css={{ display: 'flex', height: '100%' }} as="form">
```
Comment (alexdanilowicz) [inline packages/core/src/components/Onboarding/pages/ExportSettings.tsx:38]: When main is merged, this diff should not show anymore

**Pull request closed without merging**